data/mods/BrightNights/vehicle_groups.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "id": "city_vehicles",
+    "type": "vehicle_group",
+    "vehicles": [
+      [ "car_atomic", 1000 ],
+      [ "car_sports_atomic", 500 ],
```

Comment:
Sports cars have the enhanced engines and those would be better off staying rare - that way they'd be more of a reward than "oh, another enhanced engine".